config/build_settings.json

```diff
@@ -26,7 +26,7 @@
 			],
 
 			"test_includes": ["../test/pic12/gpsim_logging"],
-			"xc8_flags": ["-G", "--addrqual=require", "--summary=default,+psect,-class,+mem,-hex,-file", "--output=default,-inhx032", "--asmlist", "-P", "--mode=pro", "--opt=all,-asmfile" ]
+			"xc8_flags": ["-G", "--addrqual=require", "--summary=default,+psect,-class,+mem,-hex,-file", "--output=default,-inhx032", "--asmlist", "-P", "--mode=pro", "--opt=all,-asm,-asmfile" ]
```

Comment:
I think this change should be in the gsm_module json file so that it doesn't affect all of the other modules.  The line can be the same and should work if you just move it into the local gsm_module json file at root level under the { }